### DISCUSSION TITLE
Anyone using Riverpod in production? (Yes finally someone asked it)

### DISCUSSION THREAD

**Discussion by realshovanshah** [rrousselGit/riverpod#1157] [Q&A]
Can we pin a issue prompting the use of Riverpod in production? Seeing this used in a full fledged app can help other devs do the same by answering a lot of the questions.

Similar to and many others:
https://github.com/felangel/bloc/issues/139

**Comment by scopendo** [comment]
One (B2B) app in production and another on its way.

**Comment by Shvet** [comment]
I have one app with flutter_riverpod too

**Comment by msarkrish** [comment]
Created one issue in riverpod, like provider and bloc package
https://github.com/rrousselGit/river_pod/issues/1345
